nixos/modules/services/networking/nordvpn.nix

```diff
@@ -0,0 +1,50 @@
+{ config, lib, pkgs, ... }:
+
+with lib; {
+  options.services.nordvpn.enable = mkOption {
+    type = types.bool;
+    default = false;
+    description = mdDoc ''
+      Whether to enable the NordVPN daemon. Note that you'll have to set
+      `networking.firewall.checkReversePath = false;`, add UDP 1194
+      and TCP 443 to the list of allowed ports in the firewall and add your
+      user to the "nordvpn" group (`users.users.<username>.extraGroups`).
+    '';
+  };
+
+  config = mkIf config.services.nordvpn.enable {
+    environment.systemPackages = [ pkgs.nordvpn ];
+
+    users.groups.nordvpn = {};
+    systemd = {
+      services.nordvpn = {
+        description = "NordVPN daemon.";
+        serviceConfig = {
+          ExecStart = "${pkgs.nordvpn}/bin/nordvpnd";
+          NonBlocking = true;
+          KillMode = "process";
+          Restart = "on-failure";
+          RestartSec = 5;
+          RuntimeDirectory = "nordvpn";
+          RuntimeDirectoryMode = "0750";
```

Comment:
hmm, you're right. Both the service and `tmpfiles` rule are taken directly from the files on the original .deb package. I'll remove the rule since it appears everything runs fine without it.